src/pages/VenuePage/VenuePage.tsx

```diff
@@ -116,8 +113,6 @@ const VenuePage: React.FC = () => {
     setLocationData({ userId, locationName: venueName });
   }, [userId, venueName]);
 
-  useTitle(`${PLATFORM_BRAND_NAME} - ${venueName}`);
```

Comment:
Specifically, https://github.com/sparkletown/sparkle/pull/1396#pullrequestreview-660288190 is referring to the removal of this code and similar that was added in https://github.com/sparkletown/sparkle/pull/1389

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/026b25922801214655c53f5ddac1ebdbf802d514